### PR TITLE
Update compound v3 market list

### PR DESCRIPTION
**Pull request by MishaShWoof**

### CODE REVIEW

**Comment by dmitriy-woof-software** [comment]
Hey @slasher125. Can you review PR please? Thanks.

File: src/adaptors/compound-v3/index.js [hunk 1]
```diff
@@ -68,6 +92,30 @@ const markets = [
...
+    address: '0x2e44e174f7D53F0212823acC11C01A11d58c5bCB',
+    symbol: 'cUSDCv3',
+    underlying: '0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85',
+    underlyingSymbol: 'USDC',
+    rewardToken: '0x7e7d4467112689329f7E06571eD0E8CbAd4910eE',
+    chain: 'optimism',
+  },
+  {
+    address: '0x995E394b8B2437aC8Ce61Ee0bC610D617962B214',
+    symbol: 'cUSDTv3',
```
Comment (slasher125) [inline src/adaptors/compound-v3/index.js:113]: pls add a suffix here, `-2`, tests are currently failing cause of a duplicated id

Comment (dmitriy-woof-software) [reply src/adaptors/compound-v3/index.js:113]: @slasher125 how to update this API? We cannot receive data for Optimism and Scroll COMP data
https://github.com/DefiLlama/yield-server/blob/920f86f0a9c1b16d17e6e98ee0ced6ea6481385c/src/adaptors/compound-v3/index.js#L170

Comment (dmitriy-woof-software) [reply src/adaptors/compound-v3/index.js:113]: And for new markets. Can you help us where should we add it

Comment (dmitriy-woof-software) [reply src/adaptors/compound-v3/index.js:113]: Is this one? https://github.com/DefiLlama/defillama-server/tree/master/coins

Comment (slasher125) [reply src/adaptors/compound-v3/index.js:113]: we have just updated the price data for COMP on both OP and Scroll (https://github.com/DefiLlama/defillama-server/pull/7440). prices will be available ~1h

**Comment by llamatester** [comment]
Error while running compound-v3 adapter: 
        
 
 Test Suites: 1 failed, 1 total
    Tests:       1 failed, 395 passed, 396 total
    Snapshots:   0 total
    Time:        0.406 s
    Ran all test suites.
    
    Nb of pools: 63
     
    
    Sample pools:
    ┌─────────┬─────────────────────────────────────────────────────────────┬──────────┬────────────┬───────────────┬────────────────────┬─────┬──────────────────────────────────────────────────┬────────────────────┬───────┬─────────────┬────────────┬────────────────────┬────────────────────┬──────────────────────────────────────────────────┬────────────────────┬─────────────────────┬────────────────────┐
    │ (index) │                            pool                             │  symbol  │   chain    │    project    │       tvlUsd       │ apy │                 underlyingTokens                 │   totalSupplyUsd   │  ltv  │  poolMeta   │ borrowable │      apyBase       │     apyReward      │                   rewardTokens                   │   apyBaseBorrow    │   apyRewardBorrow   │   totalBorrowUsd   │
    ├─────────┼─────────────────────────────────────────────────────────────┼──────────┼────────────┼───────────────┼────────────────────┼─────┼──────────────────────────────────────────────────┼────────────────────┼───────┼─────────────┼────────────┼────────────────────┼────────────────────┼──────────────────────────────────────────────────┼────────────────────┼─────────────────────┼────────────────────┤
    │    0    │    '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599-cUSDCv3'     │  'WBTC'  │ 'Ethereum' │ 'compound-v3' │ 450045772.1468911  │  0  │ [ '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599' ] │ 450045772.1468911  │  0.8  │ 'USDC-pool' │   false    │                    │                    │                                                  │                    │                     │                    │
    │    1    │    '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2-cUSDCv3'     │  'WETH'  │ 'Ethereum' │ 'compound-v3' │ 428187454.1402251  │  0  │ [ '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2' ] │ 428187454.1402251  │ 0.825 │ 'USDC-pool' │   false    │                    │                    │                                                  │                    │                     │                    │
    │    2    │    '0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0-cWETHv3'     │ 'wstETH' │ 'Ethereum' │ 'compound-v3' │ 115277176.66199197 │  0  │ [ '0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0' ] │ 115277176.66199197 │  0.9  │ 'ETH-pool'  │   false    │                    │                    │                                                  │                    │                     │                    │
    │    3    │        '0xc3d688B66703497DAA19211EEdff47f25384cdc3'         │  'USDC'  │ 'Ethereum' │ 'compound-v3' │  73211672.3335495  │     │ [ '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48' ] │ 555831631.3467809  │   0   │ 'USDC-pool' │    true    │ 6.512132612087999  │ 0.4529524874105232 │ [ '0xc00e94Cb662C3520282E6f5717214004A7f26888' ] │  8.7328086208128   │ 0.24528558711496884 │ 482619959.0132314  │
    │    4    │        '0xA17581A9E3356d9A858b789D68B4d866e593aE94'         │  'ETH'   │ 'Ethereum' │ 'compound-v3' │ 51555784.14909263  │     │ [ '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' ] │ 165670267.65926093 │   0   │ 'ETH-pool'  │    true    │  1.2742883845056   │ 0.704486095477429  │ [ '0xc00e94Cb662C3520282E6f5717214004A7f26888' ] │ 1.9643263390656003 │ 0.29221882248592285 │ 114114483.5101683  │
    │    5    │    '0xc00e94Cb662C3520282E6f5717214004A7f26888-cUSDCv3'     │  'COMP'  │ 'Ethereum' │ 'compound-v3' │ 36741222.351752535 │  0  │ [ '0xc00e94Cb662C3520282E6f5717214004A7f26888' ] │ 36741222.351752535 │ 0.65  │ 'USDC-pool' │   false    │                    │                    │                                                  │                    │                     │                    │
    │    6    │    '0x9c4ec768c28520B50860ea7a15bd7213a9fF58bf-arbitrum'    │  'USDC'  │ 'Arbitrum' │ 'compound-v3' │ 24310067.795945477 │     │ [ '0xaf88d065e77c8cC2239327C5EDb3A432268e5831' ] │ 41904984.78759107  │   0   │ 'USDC-pool' │    true    │ 3.1490734613184004 │ 0.6763978114688642 │ [ '0x354A6dA3fcde098F8389cad84b0182725c6C91dE' ] │  4.9975709232384   │ 0.7580916696707373  │ 17594916.991645593 │
    │    7    │ '0x2f2a2543B76A4166549F7aaB2e75Bef0aefC5B0f-cUSDCv3-native' │  'WBTC'  │ 'Arbitrum' │ 'compound-v3' │  18213578.8098797  │  0  │ [ '0x2f2a2543B76A4166549F7aaB2e75Bef0aefC5B0f' ] │  18213578.8098797  │ 0.75  │ 'USDC-pool' │   false    │                    │                    │                                                  │                    │                     │                    │
    │    8    │    '0x514910771AF9Ca656af840dff83E8264EcF986CA-cUSDCv3'     │  'LINK'  │ 'Ethereum' │ 'compound-v3' │ 17314560.981922645 │  0  │ [ '0x514910771AF9Ca656af840dff83E8264EcF986CA' ] │ 17314560.981922645 │ 0.79  │ 'USDC-pool' │   false    │                    │                    │                                                  │                    │                     │                    │
    │    9    │    '0x1BFD67037B42Cf73acF2047067bd4F2C47D9BfD6-cUSDCv3'     │  'WBTC'  │ 'Polygon'  │ 'compound-v3' │ 15526367.70862693  │  0  │ [ '0x1BFD67037B42Cf73acF2047067bd4F2C47D9BfD6' ] │ 15526367.70862693  │ 0.75  │ 'USDC-pool' │   false    │                    │                    │                                                  │                    │                     │                    │
    └─────────┴─────────────────────────────────────────────────────────────┴──────────┴────────────┴───────────────┴────────────────────┴─────┴──────────────────────────────────────────────────┴────────────────────┴───────┴─────────────┴────────────┴────────────────────┴────────────────────┴──────────────────────────────────────────────────┴────────────────────┴─────────────────────┴────────────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama
    }

**Comment by slasher125** [comment]
@MishaShWoof pricing works now, but we need to address the pool field (see my comment)

**Pull request merged**